build.gradle

```diff
@@ -1,13 +1,15 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
+    apply from: 'versions.gradle'
```

Comment:
For consistency across modules I will be placing build and dependency versions in this new `versions.gradle` file and applying it to the build process at the top level